### PR TITLE
[Ready] Digi Leg Sprites for all Uniforms

### PR DESCRIPTION
**Pull request by Ziiro**
:cl: Quantum_m & Ziiro
add: All uniforms now have digi leg alternates!
/:cl:

Note: Uniforms with "alternate" designs (like rolling down a jumpsuit) are not compatible with this. It simply says "You adjust the style" and nothing changes.

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -631,7 +631,7 @@
 		if(!(DIGITIGRADE in species_traits)) //Someone cut off a digitigrade leg and tacked it on
 			species_traits += DIGITIGRADE
 		var/should_be_squished = FALSE
-		if(H.wear_suit && ((H.wear_suit.flags_inv & HIDEJUMPSUIT) || (H.wear_suit.body_parts_covered & LEGS)) || (H.w_uniform && (H.w_uniform.body_parts_covered & LEGS)))
+		if(H.wear_suit && ((H.wear_suit.flags_inv & HIDEJUMPSUIT) || (H.wear_suit.body_parts_covered & LEGS)) || (H.w_uniform && (H.w_uniform.body_parts_covered & LEGS)) || (H.w_uniform && (H.w_uniform.mutantrace_variation & NO_MUTANTRACE_VARIATION)))
```
[the remark this answers is not in the document]
Comment (Tayyyyyyy) [reply code/modules/mob/living/carbon/human/species.dm]: I think I figured it out!
```
var/obj/item/w_uniform = null
```
w_uniform isn't of clothing type, so you need to typecheck it and then typecast it to clothing, and then get `mutantrace_variation`

[the remark this answers is not in the document]
Comment (Ziiro) [reply code/modules/mob/living/carbon/human/species.dm]: Hmm. I'm going to have to divide this up anyway, so I think I know what I'm going to do. Thank you, Tay

**Comment by Ralta** [comment]
Woop! Hope this gets merged soon. Helps make Unathi and Vox feel that much more alien.

**Review by AndrewMontagne — APPROVED** [review]

**Review by AndrewMontagne — COMMENTED** [review]
Before this gets merged, though. Would it be worth seeing if the digitigrade sprites can be moved to a seperate *.dmi?

**Comment by SamHPurp** [comment]
@Ziiro so I can properly understand, can you supply a few photos.

**Comment by Ziiro** [comment]
dreamseeker_2018-06-24_15-36-08
https://gfycat.com/AccurateLinearImperialeagle
@SamHPurp

**Comment by Ziiro** [comment]
wait shit the detective broke, why did the detective break

**Comment by Ziiro** [comment]
lizard

fixed it

**Pull request merged**